modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -56,163 +52,166 @@ For example:
+appliance    1.1.1619717580-73a133d   
+kubernetes   1.16
 ```
 
 [NOTE]
 The Runtime Fabric component and appliance versions that are in use are also displayed in the *Runtime Fabrics* tab in Runtime Manager. If an upgrade is available, an "Upgrade to vx.x" message is displayed.
 
 == Determine the Most Recent Component Versions Available
-The most recent component versions are displayed in the *Downloads* page, located in Runtime Manager in the *Runtime Fabrics* tab. A link to each package’s release notes is provided to show the changes.
 
-You can also review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
+Review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] for information about the latest component versions.
 
 == Apply the Upgrades
-Before upgrading your production environment, perform the following steps on a nonproduction Runtime Fabric to verify compatibility with your environment.
 
-. Review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
-. Provision additional worker nodes as needed to maintain availability of applications when applying appliance upgrades.
+To upgrade Anypoint Runtime Fabric on VMs/Bare Metal, you:
+
+. Upgrade Runtime Fabric components
+. Upgrade the `rtfctl` command-line utility
+. Upgrade the Runtime Fabric appliance
+. Update the Mule deployments running on Runtime Fabric to the latest patch version
+. Verify Runtime Fabric is healthy
+. Run the upgrade
+. Upgrade node configurations
+
+=== Prerequisites
+
+* Before upgrading your production environment, perform the following steps on a nonproduction Runtime Fabric to verify compatibility with your environment. Plan to upgrade when your team is not deploying new applications or managing existing applications on Runtime Fabric.  
+* Review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
+* Provision additional worker nodes as needed to maintain availability of applications when applying appliance upgrades.
 +
 If the amount of available resources for application deployments is less than the resources of one worker node, add another worker node with the same amount of CPU, memory, and disk resources to Runtime Fabric.
 +
 See xref:manage-nodes.adoc[How to Add a Node to Runtime Fabric].
-. To upgrade Runtime Fabric components:
 
-.. Plan to upgrade when your team is not deploying new applications or managing existing applications on Runtime Fabric.  
-.. Navigate to your Runtime Fabric by selecting it on the *Runtime Fabrics* tab in Runtime Manager.
-.. Select "Upgrade to vx.x". The status of the Runtime Fabric is updated to show that the upgrade procedure is in progress.
-.. Wait for the status to transition to *Active*. 
+=== Upgrade Runtime Fabric Components
+
```

Comment:
ditto DOCS-11300 add topic for upgrading RTF-SMK #385
comments about performing actions in the sandbox vs. production